src/plots/gl3d/layout/layout_attributes.js

```diff
@@ -72,6 +72,16 @@ module.exports = {
                 'of this scene.'
             ].join(' ')
         }),
+        ortho: extendFlat({
```

Comment:
`geo` subplots set their projection via `geo?.projection.type`

https://github.com/plotly/plotly.js/blob/0714286d8c485a55bd0666f4a45f853c4f5112e5/src/plots/geo/layout/layout_attributes.js#L96-L102

3D scenes don't have equivalents for other geo attributes e.g. `projection.rotation` and `projection.parallels`, but @archmoj can you think of other potential "projection" settings we could add to 3D scenes?

Reply:
Cool attributes in `geo`! 
I was thinking something like `scale` https://github.com/plotly/plotly.js/blob/0714286d8c485a55bd0666f4a45f853c4f5112e5/src/plots/geo/layout/layout_attributes.js#L143-L153 or `fieldOfView` might be of interest later.

Reply:
Ok, so let's call this new attribute `scene.projection.type` with possible values: `'orthographic'` and `'perspective'`.

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/3550/commits/43e4a04c32025fe241a9520665c9fd46594a7bcc & https://github.com/plotly/plotly.js/pull/3550/commits/e97fc7857ff91c536aa881d95276676051ecb600.